src/primitives.es6

```diff
@@ -17,13 +17,22 @@ export const attributeElementKeys = Symbol('attributeElementKeys');
  * able to convert to and from Refract/Javascript.
  */
 export class ElementType {
-  constructor(element, content = null, meta = {}, attributes = {}) {
-    this.element = element;
+  constructor(content = null, meta = {}, attributes = {}) {
     this.meta = meta;
     this.attributes = attributes;
     this.content = content;
+  }
+
+  elementName() {
+    return 'element';
+  }
 
-    this[attributeElementKeys] = [];
+  attributeElementKeys() {
```

Comment:
FWIW, this was originally using a Symbol because it's considered private class data. For example, `Object.keys(myElement)` did not include the symbol. Now that it's a public method it will be listed there. Probably not a huge deal, but it's a side effect of this change to keep in mind.